password/password.cabal

```diff
@@ -46,6 +74,7 @@ library
     , bytestring  >= 0.10.8.1 && < 0.11
     , cryptonite  >= 0.15.1   && < 0.28
```

Comment:
It looks like there is a new release of `cryptonite`: https://github.com/commercialhaskell/stackage/issues/5849

Could you maybe bump this as well?

Reply:
done